frontend/unit/setup.ts

```diff
@@ -38,6 +38,7 @@ export const apiCall = async (): Promise<void> => {
    */
   await sleep();
   await sleep();
+  await sleep(10);
```

Comment:
Perhaps a comment here about why it's 10 seconds, in addition to the two non-parameterized calls above, would be nice.

Reply:
Unfortunately, the 10ms is arbitrary and I really don't know why the error is happening. I can't replicate it locally.

I'll add a generic comment to this effect, also leave this PR open a little longer (maybe til next week) to keep investigating.